src/iotjs_util.c

```diff
@@ -27,7 +27,10 @@
 
 iotjs_string_t iotjs_file_read(const char* path) {
   FILE* file = fopen(path, "rb");
-  IOTJS_ASSERT(file != NULL);
+  if (file == NULL) {
+    iotjs_string_t empty_content = iotjs_string_create();
+    return empty_content;
+  }
```

Comment:
@haesik  For testcase, adding following case to `test_process_readsource.js` would be enough to cover your change. It causes segmentation fault.

```
process.readSource('notExist');
```

Although I still think we need to distinguish them (not existing source vs. existing and empty content), the fix to segmentation fault itself is good.